python/tako/client/task.py

```diff
@@ -1,8 +1,8 @@
 from typing import List, Optional
 
 import msgpack
-from tako.client.program import ProgramDefinition
-from tako.client.resources import ResourceRequest
+from python.tako.client.program import ProgramDefinition
```

Comment:
Do not change it to "python". The python is not the name of the package but just a directory where package "tako" is placed. If you want to run it locally, modify env variable PYTHONPATH (or use pytests where conftest.py do the path changes)

Reply:
Thank you, this was a mistake